src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -845,6 +845,17 @@ public IEnumerable<Document> GetDocumentsFrom(DocumentsOperationContext context,
             }
         }
 
+        public IEnumerable<Document> GetAllDocumentsFrom(DocumentsOperationContext context, long etag)
```

Comment:
We already have this method. Use 'GetDocumentsFrom` instead.

Reply:
That is used for replication and return the wrong result

Reply:
```
public IEnumerable<Document> GetDocumentsFrom(DocumentsOperationContext context, long etag, long start, long take, DocumentFields fields = DocumentFields.All)
```

Use this one.